### ISSUE TITLE
Accessibility audit of VBA Centralized Content implementation

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#14899]
**1. What is this request for?**

VBA Centralized Content type is being created for the VBA Regional Office MVP.

Review implementation in #14699 -- this is Centralized Content, as it pertains to VBA. 
* Centralized Content content type -- not a candidate for this review
* The Centralized Content node instance where Natl editors will edit info -- https://prod.cms.va.gov/centralized-content/national-vba-facilities-content
* The VBA Facility content type where that node content is pulled in -- e.g. https://prod.cms.va.gov/vba-facilities/locations/fort-harrison-va-regional-benefit-satellite-office-at-david-j-thatcher-va-clinic

### Acceptance Criteria
- [x] Code recommendations documented as ticket  
- [x] UX recommendations documented in Figma created in #14889

**Comment by laflannery** [comment]
On the centralized Content itself there are some things that might not matter because this might not be real content yet but just in case:
1. The title doesn't match the convention specified in the help text:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/fe64c92f-be17-400e-ae18-54dc7a306b12)
2. Possibly a typo in the purpose:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/5e90ca6d-e215-4bf3-96a8-be6eec13fff1)
3. Typo in the Link text for the Twitter link (missing the V):
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/4cde0ed5-8c8a-46cf-ac98-d4cc1c5fcf49)



These are not accessibility specifically but I am confused about the relationship between the Centralized content and the VBA Facility:
1. Benefit office overview
   1. The description does not accurately reflect where this displays. I feel like this might be ok though because this is not real content yet?
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0382925e-9ea3-4cc1-8bc6-8bcb68ce68fa)
5. The order of the centralized content sections doesn't seem to follow the order they appear on the Facility content type which seems confusing to me:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/d611932d-1fb1-4eb6-bc07-647d33fd7d9f)
6. The scope title field display is inconsistent - sometimes it shows exactly as it was entered within the Centralized content, sometimes it shows but not quite exactly as entered, sometimes it doesn't show at all.
   1. Exactly as entered
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/44d131e5-c698-489e-8d5f-4d4923f22bf8)
   2. Showing but not exactly as entered
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/15bb61b2-326f-41b2-a86e-1d369638bd5f)
   3. Not shown at all
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/8d619c2f-e66d-4158-ad97-dfeebbf5fd70)
7. The tooltip for the Benefits hotline section says "International editors" instead of "National editors". That seems possibly not correct?
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/cabc001a-827b-4d71-af9b-573c48babf57)

**Comment by laflannery** [comment]
Tickets created (nothing specifically because of this work but just new issues I found because I was reviewing this):
- #15662
- #15661

**Comment by davidmpickett** [comment]
> 5. "Can't find benefits" content:
>    1. "ET" should not have periods in between the letters according to VA content guidelines
>    2. Phone numbers should be linked to be able to be dialed easily.
>    3. "Or" is currently capitalized but it is not the start of a new sentence so should not be.

Tagging in @mmiddaugh for visibility since these are content edits and don't require engineer involvement. Here's a recap of changes I made as a Content Admin.

1) ET was formatted correctly. It's at the end of a sentence so there's a period after the T, but there's no period between E & T. BUT, then I discovered that the [content guide](https://design.va.gov/content-style-guide/dates-and-numbers#times-and-time-zones) specifically recommends formatting these sentences differently to avoid having the time zone at the end, so I changed that.
2) I used the [Aria label tool](https://prod.cms.va.gov/help/vamc/aria-label-tool-for-phone-number-accessibility) to generate the tags for the phone number.
3) I think we could probably remove this sentence entirely -> "Or find another VA benefits location near you." And maybe we update the call to action link text to "Find a VA benefits location **near you**" 

![Screenshot 2023-10-13 113833](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/719a0efc-c93e-4f59-84cf-52c20af29d0d)

**Comment by davidmpickett** [comment]
@omahane tagging you in on responding to these items as they are configuration changes.

> 8. The scope title field display is inconsistent - sometimes it shows exactly as it was entered within the Centralized content, sometimes it shows but not quite exactly as entered, sometimes it doesn't show at all.

I'm translating this to mean: the label of an entity fetch field should match the label of the scope field it's fetching from. I created a [quick branch to update the two easiest examples of this](https://github.com/department-of-veterans-affairs/va.gov-cms/compare/VACMS-14899-Update-VBA-CC-fetch-field-labels?diff=unified).

The National spotlight section will require more work, because we need to contextualize those differently and explain the relationship between national and local spotlights. The UX of this part will likely be addressed in [future ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14888) that could then guide configuration updates

> 9. The tooltip for the Benefits hotline section says "International editors" instead of "National editors". That seems possibly not correct? This is only on the edit screen not the View (View says National)

This is also fixed in my branch

**Comment by davidmpickett** [comment]
@thejordanwood gets 🥝 comments as they are UX tasks. 

> 6. 🥝 Benefit office overview - The description does not accurately reflect where this displays. I feel like this might be ok though because this is not real content yet?
> ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0382925e-9ea3-4cc1-8bc6-8bcb68ce68fa)

These description fields are a known area that needs love: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14978

> 7. 🥝 The order of the centralized content sections doesn't seem to follow the order they appear on the Facility content type which seems confusing to me:

Might be good to address the desired order of fields on VBA Facility content type holistically on [this future ticket ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14888)

**Comment by davidmpickett** [comment]
> 2. 🍇 Possibly a typo in the purpose:

😅This one is totally on me. I gave @omahane this exact text in the specification and he input it as written. This field is locked to true admins so I can't fix this myself. @mmiddaugh or @omahane, could one of you fix that to be:

**Purpose:** This content displays ~~used~~ on all regional & satellite benefit office pages.

**Comment by davidmpickett** [comment]
🍉is for ?

> 1. 🍉 The title doesn't match the convention specified in the help text:

The Vet Center centralized content is also breaking this convention. Is this convention a requirement or a suggestion? Who on CMS Platform (as owners of Centralized Content) is governing this convention?

**Comment by davidmpickett** [comment]
> 4. 🍇 "Learn more" tends to imply link text. We don't have a FE yet so I can't tell if used here is clear how folks are supposed to Learn more but it definitely makes me question it and maybe we would want to use different words here:

This sentence isn't really adding any value to the description, so I suggest we cut it altogether.

@mmiddaugh FYI here's a quick link to see all the 🍇 edits I made https://prod.cms.va.gov/node/61744/revisions/view/813608/816462/visual_inline

**Comment by thejordanwood** [comment]
@davidmpickett @laflannery 

> 🍇"Can't find benefits" content:
> "ET" should not have periods in between the letters according to VA content guidelines
> Phone numbers should be linked to be able to be dialed easily.
> "Or" is currently capitalized but it is not the start of a new sentence so should not be.

> ❓ I think we could probably remove this sentence entirely -> "Or find another VA benefits location near you." And maybe we update the call to action link text to "Find a VA benefits location near you"

This wording was suggested by CAIA. ([Sketch comment](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff/p/FC995695-308A-4493-8461-D9AFE6DC6B25/canvas?annotation=a1f297c5-a12e-4acb-ae38-0d6f0aaec3fd))

> 🍇 "Learn more" tends to imply link text. We don't have a FE yet so I can't tell if used here is clear how folks are supposed to Learn more but it definitely makes me question it and maybe we would want to use different words here:

> This sentence isn't really adding any value to the description, so I suggest we cut it altogether.

@mmiddaugh You're correct! This wording was also suggested by CAIA. ([Another Sketch comment](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff/p/FC995695-308A-4493-8461-D9AFE6DC6B25/canvas?annotation=e7f6d52e-1d66-4562-a154-9c2428ffeb50))

**Comment by laflannery** [comment]
@davidmpickett Can I close this? The ticket is done but the comments are confused - I feel like if stuff in the comments should be done it should be moved into new tickets?

**Comment by davidmpickett** [comment]
@laflannery Technically that's up to @xiongjaneg, but it's definitely past the point where you need to do anything. Moving to PO review

**Comment by xiongjaneg** [comment]
@xiongjaneg to review and stub out subsequent tickets then close

**Comment by xiongjaneg** [comment]
scheduled meeting with CMS Team and Facilities product folks to discuss ^ on Oct. 30
